pkg/client/dtclient/settings_client_test.go

```diff
@@ -44,7 +46,7 @@ func TestNewClassicSettingsClient(t *testing.T) {
 
 		client, err := NewClassicSettingsClient(corerest.NewClient(server.URL(), server.Client()))
 		assert.NoError(t, err)
-		assert.Equal(t, settingsSchemaAPIPathClassic, client.settingsSchemaAPIPath)
+		assert.Equal(t, "/api/v2/settings/schemas", client.settingsSchemaAPIPath)
```

Comment:
What was the reason for replacing it with the actual value?
On the one side yes, tests should be rather static and not depend on such values but on the other side, why just this one and not the others like `settingsObjectAPIPathClassic`